packages/grafana-data/src/types/theme.ts

```diff
@@ -243,6 +243,9 @@ export interface GrafanaTheme extends GrafanaThemeCommons {
     formCheckboxBgChecked: string;
     formCheckboxBgCheckedHover: string;
     formCheckboxCheckmark: string;
+
+    cardBackground: string;
+    cardBackgroundHover: string;
```

Comment:
use bg2  (it is the card barckground), and then styleMixins.hoverColor(bg2, theme)

Reply:
Or if you can use the Card component